apps/web/src/components/WhyBase/ActiveCommunityOfBuilders.tsx

```diff
@@ -0,0 +1,130 @@
+import Link from 'apps/web/node_modules/next/link';
+import Image from 'apps/web/node_modules/next/image';
+import { StaticImport } from 'apps/web/node_modules/next/dist/shared/lib/get-img-props';
+
+import { ButtonWithLinkAndEventLogging } from 'apps/web/src/components/Button/ButtonWithLinkAndEventLogging';
+
+import aerodromeImage from './images/aerodrome.svg';
+import doodlesImage from './images/doodles.svg';
+import morphoImage from './images/morpho.svg';
+
+import { partners, Partner } from 'apps/web/src/components/WhyBase/partners';
+
+export default async function ActiveCommunityOfBuilders() {
+  return (
+    <div
+      id="communityOfBuilders"
+      className="mt-10 flex w-full max-w-[1440px] flex-col px-12 pb-6 sm:mt-16 sm:px-16 sm:pb-8 lg:mt-20 lg:px-24 lg:pb-10"
+    >
+      <div className="flex flex-col justify-between md:grid md:grid-cols-[1fr_1fr]">
+        <div className="mr-8 hidden md:block">
+          <CommunityPartners communityPartners={partners} />
+        </div>
+        <div className="flex grow flex-col justify-start space-y-6 lg:mx-8 lg:justify-around xl:mx-20">
+          <h2 className="flex flex-row font-display text-3xl sm:text-4xl lg:text-6xl">
+            <span>1.</span>
+            <span className="ml-4">Join an active community of Builders</span>
+          </h2>
+          <div className="md:hidden">
+            <CommunityPartners communityPartners={partners} />
+          </div>
+          <span className="text-base text-white sm:text-lg">
+            Join a community of thousands builders just like you, building some of the coolest
+            projects onchain. Reach out to our Discord support team for help.
+          </span>
+          <div>
+            <ButtonWithLinkAndEventLogging
+              href="https://discord.com/invite/buildonbase"
+              eventName="join_discord"
+              target="_blank"
+              rel="noreferrer noopener"
+              linkClassNames="inline-block"
+              buttonClassNames="uppercase font-mono font-medium"
+            >
+              Join the Discord
+            </ButtonWithLinkAndEventLogging>
+          </div>
+        </div>
+      </div>
+      <div className="mt-10 sm:mt-12 lg:mt-16">
+        <span className="text-lg sm:text-xl">Community Spotlight</span>
+        <div className="mt-4 flex flex-col flex-wrap justify-between gap-4 sm:flex-row">
+          {communityCards?.map((card) => (
```

Comment:
I think communityCards is always defined? if so no need for ?.

Reply:
sg, all set